include/grpc++/server_builder.h

```diff
@@ -170,6 +178,28 @@ class ServerBuilder {
     int* selected_port;
   };
 
+  struct SyncServerSettings {
+    SyncServerSettings()
+        : num_cqs(GPR_MAX(gpr_cpu_num_cores(), 4)),
+          min_pollers(1),
+          max_pollers(INT_MAX),
+          cq_timeout_msec(1000) {}
+
```

Comment:
No particular reason. Actually that is how I did it in a previous iteration of the code. But after going back and forth a little, it just felt like there is no need to expose a new structure to the client using server_builder.